pages/today.vue

```diff
@@ -0,0 +1,11 @@
+<script setup lang="ts">
+import { ref } from 'vue';
+
+const date = ref(Date.now());
```

Comment:
If you're not going to change this `date` value, no need to wrap it with `ref`. You could leave it as a static date variable.